src/main.rs

```diff
@@ -212,9 +216,9 @@ async fn handle_action(
     debug!("{}", &action.action);
     match action.action.as_str() {
         "version" => Response::result(6),
-        "deckNames" => Response::result(["jpdb"]),
-        "modelNames" => Response::result(["jpdb"]),
-        "modelFieldNames" => Response::result(["word", "reading", "sentence"]),
+        "deckNames" => Response::result(["jpdbx"]),
```

Comment:
not sure i understand what's going on here.

with this change, i think it will require every user to mess with yomichan to change the field name, in which case could they not just remove the `jpdb` thing and re-add it in some way? perhaps by turning ankiconnect off and back on?

Reply:
Turning ankiconnect off and on didn't work for me, I couldn't find any other way to force it to refresh the fields list.
At fist I thought it was fine because to use the definition field you would have to edit your anki settings anyway.
But this affects all users even if they don't want to use definitions... my bad

Reply:
I think yomichan doesn't ask for new fields unless you change the model type. I just created a dummy model that can be used to force a refresh which should fix this issue in [4f8450c](https://github.com/kampffrosch94/jpdb-connect/pull/16/commits/4f8450c342bb02d79598350f20875bb8014b33ff)